transformations/bigquery/index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Google BigQuery Transformation
+permalink: /transformations/bigquery/
+---
+
+* TOC
+{:toc}
+
+[BigQuery](https://cloud.google.com/bigquery) offers a range of features:
+
+- Fully managed, serverless data warehouse
+- Automatic scaling of compute resources
+- Storage and analysis of multi-terabyte datasets
+- High-speed streaming insertion of data
+- Integrates with Google's data analytics ecosystem
+
+## Limits
```

Comment:
tady jsem to ted popisoval, jake to ma limity jeste ze strany kebooly https://help.keboola.com/storage/byodb/#bigquery-limitations